### ISSUE TITLE
tcpevents.sock: bind: address already in use

### ISSUE THREAD

**Issue by nkinkade** [m-lab/tcp-info#126]
The tcp-info pod on mlab4-fra04 was crash looping because of this error

```
main.go:101: Could not listen on%!(EXTRA string=/var/local/tcpinfoeventsocket/tcpevents.sock) (error: listen unix /var/local/tcpinfoeventsocket/tcpevents.sock: bind: address already in use)
```

This was supposed to have been fixed in PR #124. But I believe this is now the 2nd time we have seen this error crop back up despite the "fix". We need to investigate why this could still be happening.

**Comment by nkinkade** [comment]
Looking at PR #124. It doesn't look like a fix at all. If I had  to guess, this was a vscode auto-completion mistake that slipped past review. It should be a call to `os.Remove()`, not `os.Readlink()`.

**Comment by nkinkade** [comment]
The bug fix from #127 was integrated into a new release which has been promoted to production.
